### PR TITLE
New package: aspell-cs-20040614.1

### PR DESCRIPTION
**Pull request by PaperMountainStudio**

### CODE REVIEW

File: srcpkgs/aspell-cs/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'aspell-cs'
+pkgname=aspell-cs
+version=20040614
+revision=1
+archs=noarch
+wrksrc="aspell6-cs-${version}-1"
```
Comment (Chocimier) [inline srcpkgs/aspell-cs/template]: Maybe version=20040614.1 and ${version/./-}  ?

File: srcpkgs/aspell-cs/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+build_style=configure
+hostmakedepends="aspell-devel"
+short_desc="Czech dictionary for aspell"
+maintainer="Michal V<michal@pmsdev.tk>"
```
Comment (Chocimier) [inline srcpkgs/aspell-cs/template]: Add space after last name.

File: srcpkgs/aspell-cs/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+license="GPL-2"
```
Comment (Chocimier) [inline srcpkgs/aspell-cs/template]: GPL-2.0-only

**Comment by PaperMountainStudio** [comment]
Thank you for your review. I hope I improved the template as you wanted.

**Comment by Chocimier** [comment]
Template looks fine but please include version in commit titles, see git log or Manual.md

**Comment by PaperMountainStudio** [comment]
Sorry, I missed that one.

**Pull request merged**